README.md

```diff
@@ -1,3 +1,5 @@
 # microprofile-starter
 
 [![Gitter](https://badges.gitter.im/eclipse/microprofile-starter.svg)](https://gitter.im/eclipse/microprofile-starter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+[MicroProfile starter - Generate MicroProfile Maven Project](http://mp-starter-jessie.1d35.starter-us-east-1.openshiftapps.com/mp-starter/main.xhtml;jsessionid=ZfUVsyTf581Ma3eyA-ZI8tt8G9NYNa_DnW9tJUBt.mp-starter-1-5mp87?dswid=-1840)
```

Comment:
@Daniel-Dos Why would you include the session id in the link? The session expired already anyway and it makes the link to look suspicious.

Reply:
hi
I had not even noticed it, thank you very much for reporting, I'm going to withdraw.

You're right, this may even be suspect.

Reply:
done.

Reply:
@Daniel-Dos While I was playing with #9 I set it up here https://microstarter.foci.life  (don't mind the site, it is just my private playground)
Although I don't get the "Splashscreen" at the beginning. It is not loading anything, the context is ready. In the end it is much more convenient to go directly to https://microstarter.foci.life/main.xhtml 

It should be fixed, nobody should be forced to wait for a pointless splashscreen IMHO.

Reply:
very good . 

I agree, SplashScreen is useless.
but for me also the splashscreen does not appear, it only appears if you remove main.xhtml.

What suggestion do you have? I leave the pr as is, do I include your link?
thank you .